integration-tests/src/test/java/org/apache/fineract/integrationtests/FixedDepositTest.java

```diff
@@ -153,9 +153,13 @@ public void testFixedDepositInterestCalculationWithWrongCompoundingPeriod() {
         JsonElement element = parser.parse(apiRequestBodyAsJson);
         fixedDepositAccountInterestCalculationServiceImpl = new FixedDepositAccountInterestCalculationServiceImpl(
                 new DepositAccountDataValidator(new FromJsonHelper(), null), new FromJsonHelper());
-
-        assertThrows(PlatformApiDataValidationException.class, () -> fixedDepositAccountInterestCalculationServiceImpl
```

Comment:
This is also valid, but assertThrows can return the exception object which was raised as well and you can assert it, but like i said. this is fine also ;)